tests/resources/OVA-Util.robot

```diff
@@ -42,27 +42,54 @@ ${tls_not_disabled}  False
 *** Keywords ***
 # Requires vc credential for govc
 Deploy VIC-OVA To Test Server
-    [Arguments]  ${dhcp}=False  ${build}=False  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}  ${datastore}=%{TEST_DATASTORE}  ${cluster}=%{TEST_RESOURCE}  ${datacenter}=%{TEST_DATACENTER}
-    Run Keyword if  ${build}  Build Unified OVA
-    ${rev}=  Run  git rev-parse --short HEAD
-    Set Test Variable  ${ova_path}  bin/vic-1.1.0-${rev}.ova
+    [Arguments]  ${dhcp}=True  ${protocol}=https  ${build}=False  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL_ARRAY}  ${datastore}=%{TEST_DATASTORE}  ${cluster}=%{TEST_RESOURCE}  ${datacenter}=%{TEST_DATACENTER}
+    Set Global Variable  ${ova_path}  bin/vic-v1.1.0-bf760ea2.ova
```

Comment:
This is just for my local testing, will be using the ${rev} when merging the PR.